Cannot run `yarn start`

After building docker image, running docker image results in the following error:


```typescript
[0] /usr/src/app/node_modules/ts-node/src/index.ts:434
[0]     return new TSError(diagnosticText, diagnosticCodes)
[0]            ^
[0] TSError: ⨯ Unable to compile TypeScript:
[0] src/resolvers/index.ts(1,27): error TS2307: Cannot find module '../schema/__generated__/graphql' or its corresponding type declarations.
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -11,6 +11,8 @@ RUN yarn install --pure-lockfile --non-interactive
 
 COPY . .
 
+RUN yarn generate
+
 EXPOSE 3000
 EXPOSE 4000
 

--- a/package.json
+++ b/package.json
@@ -9,6 +9,7 @@
   "scripts": {
     "client": "yarn workspace client start",
     "server": "yarn workspace server start",
+    "generate": "yarn workspace server generate && yarn workspace client generate",
     "start": "concurrently --kill-others-on-fail \"yarn server\"  \"yarn client\"",
     "dev": "concurrently --kill-others-on-fail \"yarn workspace client dev\" \"yarn workspace server dev\"",
     "format": "prettier --write .",

